cdap-cli/src/main/java/co/cask/cdap/cli/util/table/AltStyleTableRenderer.java

```diff
@@ -15,9 +15,11 @@
  */
```

Comment:
This code should not be a part of this PR; agreed.  I had pulled in that fix to my builds, but did not intend to include it in my PR.  I may need help remedying this situation as I'm somewhat new to git.